src/pages/foundation.astro

```diff
@@ -0,0 +1,51 @@
+---
+import Layout from "../layouts/Layout.astro"
+import Heading from "../components/Heading.astro"
+import PostBody from "../components/PostBody.astro"
+
+const title = `DDEV Foundation`
+---
+
+<Layout
+    title={title}
+>
+  <main class="max-w-4xl mx-auto mb-24">
+    <Heading title={`DDEV Foundation`} />
+    <div class="lg:flex">
+      <div class="px-6 lg:px-0 lg:w-2/3">
+        <PostBody>
+          <p>
+            The DDEV Foundation, a Colorado (USA) nonprofit corporation, is the developer of [DDEV](https://ddev.readthedocs.io). It was formerly named the Localdev Foundation. It serves as the fiscal entity for the project and owns the bank account, etc.
```

Comment:
Two issues with this link:

1. This is plain markup, so a Markdown link won’t work. (You can see that on the page.)
2. I would argue that the canonical internet home for DDEV is either ddev.com or github.com/ddev/ddev, so linking to the docs seems strange and unexpected here.

(I’d just as well point the ddev/ddev repository to ddev.com instead of the docs, since ddev.com should theoretically describe DDEV clearly and link to different interests like project support, technical support, and documentation. Same thing I mentioned earlier about intentional channeling and funneling.)

Reply:
Fixed, sorry, markdown is in my blood now and copy/paste. 

> (I’d just as well point the ddev/ddev repository to ddev.com instead of the docs, since ddev.com should theoretically describe DDEV clearly and link to different interests like project support, technical support, and documentation. Same thing I mentioned earlier about intentional channeling and funneling.)

Didn't understand your statement here, but if you were saying "We should use ddev.com as the standard reference when talking about the project" I agree.

Reply:
We’re agreeing then—I’m referencing other places where a whole bunch of links could be a simple link to ddev.com for a more clear path to other things:
![Screen Shot 2023-08-07 at 10 10 24 AM@2x](https://github.com/ddev/ddev.com-front-end/assets/2488775/f12a4834-36ef-494a-92d7-83c7841d6286)
![Screen Shot 2023-08-07 at 10 11 00 AM@2x](https://github.com/ddev/ddev.com-front-end/assets/2488775/0b3942a2-ba9c-4ca6-a3fb-28bb4076862f)

Reply:
absolutely.  Those are done.